robosherlock/src/flowcontrol/src/RSProcessManager.cpp

```diff
@@ -170,9 +172,9 @@ bool RSProcessManager::executePipelineCallback(robosherlock_msgs::ExecutePipelin
   if(req.annotators.empty())
     return false;
 
-  std::vector<std::string> newPipelineOrder = req.annotators;
+  std::vector<std::string> new_pipeline_orders[0] = req.annotators;
```

Comment:
this is a very weird thing you are doing here...mixing array declarations with std::vectors...what is this? You can probably survive with a simple vect of strings